### DISCUSSION TITLE
Add message prefix

### DISCUSSION THREAD

**Discussion by ghost** [QuickShop-Community/QuickShop-Hikari#158] [Ideas]
Add message prefix to let users to customize

**Comment by Ghost-chu** [comment]
Not a good idea.
Most messages designed for no-prefix style.
Force add a prefix for all message will kill things who need no-prefix like ControlPanel etc.

Different users have different ideas about whether a particular message should have a prefix or not.
Therefore, if you need to add a prefix, you should manually edit the language file and add a prefix to it.

I don't think there is an out-of-the-box solution here that works for everyone.
